.env.example

```diff
@@ -25,6 +25,8 @@ LOG_LEVEL=debug
 NOTIFICATIONS_EMAIL_USER="no-reply@senecacollege.ca"
 SMTP_PORT=1025
 
+NOTIFICATIONS_REPEAT_PATTERN="* */5 * * * *"
```

Comment:
I'm not sure if this is correct. I can see that this is what the BullMQ docs say, but I still don't think it is correct:

If I read this correctly, you are saying, repeat this **every second** if `(minute % 5) === 0` (minute can be devided by 5 and there is no remainder)

I'd be much happier if we used `repeat: {every: 5 * 60 * 1000}`. A lot of people don't understand the cronjob format

source: https://docs.bullmq.io/guide/jobs/repeatable
and I think their docs is incorrect
> pattern: '* 15 3 * * *',
> // Repeat job once every day at 3:15 (am)

in this case it should have been `0 15 3 * * *` ... theirs should repeat in every second during that 1 minute period ... at least according to how crond works